guides/common/modules/proc_integrating-idm-dns-with-tsig-authentication.adoc

```diff
@@ -0,0 +1,64 @@
+[id="integrating-idm-dns-with-tsig-authentication"]
+= Integrating IdM DNS with TSIG authentication
+
+You can configure an IdM server to use the secret key transaction authentication for DNS (TSIG) technology that uses the `rndc.key` key file for authentication.
```

Comment:
You can technically name it anything you want and `rndc.key` is just an example.
```suggestion
You can configure an IdM server to use the secret key transaction authentication for DNS (TSIG) technology that uses a key file for authentication.
```